packages/metrics-discovery/datadog/prometheus.d/conf.yaml

```diff
@@ -0,0 +1,10 @@
+init_config: {}
+
+instances:
+  - prometheus_url: "http://prometheus:9090/federate?match%5B%5D=%7Bjob%3D%22river-node%22%7D"
+    max_returned_metrics: 9999
+    namespace: "river-node"
+    metrics:
```

Comment:
it could be helpful to include "*process*" metrics too which correlate to the health of the stream node